fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -65,6 +65,10 @@
      */
     private TStorageFormat storageFormat = TStorageFormat.DEFAULT;
 
+    // This table has been deleted. if hasDelete is false, the BE segment must don't have deleteConditions.
```

Comment:
If FE does checkpoint or cleans up the timeout delete jobs, hasDelete will not be set, but BE tablet maybe have delete conditions.

Reply:
hasDelete will be persisted when doing checkpoint